flask_config.py

```diff
@@ -3,6 +3,8 @@
 
 class Config:
```

Comment:
This file might be worth renaming to `trello_config.py` as this is no longer used to manage the flask session cookie encryption (similarly it doesn't need to be referenced in app.py)